Kernel/Library/KString.cpp

```diff
@@ -55,6 +55,11 @@ NonnullOwnPtr<KString> KString::must_create_uninitialized(size_t length, char*&
     return KString::try_create_uninitialized(length, characters).release_value();
 }
 
+bool KString::operator==(StringView other) const
+{
+    return view() == other;
```

Comment:
Where do you need this that it's inconvenient to just call `.view()` on the KString before using `==`? I'd rather drop this commit.